content/topics/web-frontend/intro-to-web-design/_index.md

```diff
@@ -35,4 +35,11 @@ These links will teach you the basics
 
 ## Accessibility
 
-https://www.w3.org/WAI/fundamentals/accessibility-intro/
+- https://www.w3.org/WAI/fundamentals/accessibility-intro/
+
+- [A developer's guide to web design for non-designers](https://www.freecodecamp.org/news/a-developers-guide-to-web-design-for-non-designers-1f64ce28c38d/)
```

Comment:
These two resources should be under the "Principles of design" heading. Right now they are under the incorrect heading.

Reply:
Accessibility stuff should be under the accessibility heading. And design stuff should be under the design heading.
```
## Accessability
- https://www.w3.org/WAI/fundamentals/accessibility-intro/

## Principles of Design
 - [A developer's guide to web design for non-designers](https://www.freecodecamp.org/news/a-developers-guide-to-web-design-for-non-designers-1f64ce28c38d/)
- [design-101-for-developers](https://academy.realm.io/posts/christopher-downer-design-101-for-developers/)
- [7 principles of design for coders](https://learntocodewith.me/posts/7-principles-of-design/)
```

Reply:
Headings should be accurate when used